server/service/mdm.go

```diff
@@ -2444,3 +2445,259 @@ func (svc *Service) DeleteMDMAppleAPNSCert(ctx context.Context) error {
 
 	return svc.ds.SaveAppConfig(ctx, appCfg)
 }
+
```

Comment:
nit: not required to change, but `mdm.go` is for general MDM methods, this is Apple specific so ideally it goes in `apple_mdm.go`

Reply:
yeah, but it's not a big deal, we can leave everything here for now too